test/functional/tests/security/test_security_user.py

```diff
@@ -41,8 +41,8 @@ def test_user_cli():
         cache_dev = cache_dev.partitions[0]
         core_dev = TestRun.disks['core']
         core_dev.create_partitions([Size(1, Unit.GibiByte), Size(256, Unit.MebiByte)])
-        core_dev = core_dev.partitions[0]
         core_dev1 = core_dev.partitions[1]
+        core_dev = core_dev.partitions[0]
```

Comment:
What do you think about naming these variables like core_disk/cache_disk and those with assigned partitions like core_dev/cache_dev?

Reply:
Why don't just name them cache_dev/core_dev and cache_part/core_part?

Reply:
That's good idea. Done.